src/sagas/index.js

```diff
@@ -0,0 +1,63 @@
+import { takeEvery, call, put, all } from 'redux-saga/effects'
+import contentActions from '../actions/content'
+import axios from 'axios'
+
+const {
+  CONTENT_START_LOADING,
+  CONTENT_SEARCH_START,
+} = contentActions.types
+
+const {
+  fetchedContent,
+  fetchedErrorContent,
+} = contentActions.actions
+
+const API_URL = 'https://api.giphy.com/v1/gifs/'
+const API_KEY = 'LRB79OdJK91f2MQW8XLsq0oS1DZU5OmO'
```

Comment:
Use `.env` variables to save your api keys and do not expose them versioning them. You can provide en `.env.example` file expressing its usage.

https://www.npmjs.com/package/dotenv